CHANGELOG.md

```diff
@@ -6,24 +6,26 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## Unreleased
```

Comment:
Commenting out the Unreleased line gives better output in release PRs, as otherwise the rendered markdown kinda looks like the latest version is unreleased:

## Unreleased
## [3.0.0-pre.9] - 2018-01-25

Reply:
+1 doing it.  Also changing the date to 26th.